src/components/Person/Person/Person.jsx

```diff
@@ -0,0 +1,28 @@
+export const Person = ({
+  person: {
+    name,
+    age,
+    isMarried,
+    partnerName,
+    sex,
+  }
+}) => (
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+      <p className="Person__partner">
+        {isMarried
+          ? `${partnerName} is my ${sex === 'f'
+            ? `husband` : `wife`}`
```

Comment:
Try to avoid nesting ternary operators. Let's create a variable for husband/wife message
![image](https://github.com/mate-academy/react_person/assets/96409563/884ed3d2-725c-4de8-a4aa-cbf7676e543d)